airlock_processor/StatusChangedQueueTrigger/__init__.py

```diff
@@ -47,9 +52,31 @@ def main(msg: func.ServiceBusMessage):
 
     logging.info('Processing request with id %s. new status is "%s", type is "%s"', req_id, new_status, type)
 
+    try:
+        tre_id = os.environ["TRE_ID"]
+    except KeyError as e:
+        logging.error(f'Missing environment variable: {e}')
+        raise
+
+    if new_status == "draft" and request_type == constants.IMPORT_TYPE:
+        account_name = constants.STORAGE_ACCOUNT_NAME_IMPORT_EXTERNAL.format(tre_id)
+        account_rg = constants.CORE_RESOURCE_GROUP_NAME.format(tre_id)
+        sa_connection = blob_operations.get_storage_connection_string(account_name, account_rg, storage_client)
+        blob_service_client = BlobServiceClient.from_connection_string(sa_connection.connection_string)
+        blob_operations.create_container(blob_service_client, req_id)
```

Comment:
these 3 lines are identical to those on lines71-73
you can reuse it

Reply:
done